src/containers/course-section/CourseSectionContainer.tsx

```diff
@@ -45,26 +45,19 @@ import {
   CourseResources
 } from '~/types'
 import { styles } from '~/containers/course-section/CourseSectionContainer.styles'
+import CourseSectionsList from '../course-sections-list/CourseSectionsList'
```

Comment:
use import alias

Reply:
deleted this line, because it's unused now